site/content/component/mqtt-adapter.md

```diff
@@ -201,41 +201,70 @@ The same holds true analogously for the *hono-service-device-registry.hono* addr
 
 ## Using the Telemetry Topic Hierarchy
 
-The following examples use a username and password for connecting to MQTT. The username reflects the auth-id of the [Credentals API]({{< relref "api/Credentials-API.md" >}}) and usually represents a device.
+The following examples use a username and password for connecting to MQTT. The username reflects the auth-id of the [Credentals API]({{< relref "api/Credentials-API.md" >}}) and usually represents a device. It may be identical to the device-id.
 
-**NB** The standard setup of the [Device Registry component]({{< relref "component/device-registry.md" >}}) provides the example auth-id `sensor1` with the password `hono-secret` which are used here. 
+**NB** The standard setup of the [Device Registry component]({{< relref "component/device-registry.md" >}}) provides the example auth-id `sensor1` with the password `hono-secret` 
+and the device-id `4711` which are used here. Some devices may not know their device-id but only their auth-id - this case is covered in the examples as well. 
 
 
-### Upload Telemetry Data
+### Upload Telemetry Data (authenticated devices)
 
-* Topic: `telemetry/${tenantId}/${deviceId}`
-* Client-id: ${deviceId}
+* Topic: `telemetry/${tenant-id}/${device-id}` or just `telemetry` (short form)
 * Payload:
   * (required) Arbitrary payload
 
 **Example**
 
-Upload a JSON string for device `4711`:
+Upload a JSON string for device-id `4711`:
+
+    $ mosquitto_pub -u 'sensor1@DEFAULT_TENANT' -P hono-secret -t telemetry/DEFAULT_TENANT/4711 -m '{"temp": 5}'
+
+or (if the device does not know the device-id itself)
 
-    $ mosquitto_pub -i 4711 -u 'sensor1@DEFAULT_TENANT' -P hono-secret -t telemetry/DEFAULT_TENANT/4711 -m '{"temp": 5}'
+    $ mosquitto_pub -u 'sensor1@DEFAULT_TENANT' -P hono-secret -t telemetry -m '{"temp": 5}'
```

Comment:
IMHO we should use the short version (`telemetry` topic) as the example given and mention the FQTA version in a note only.

Reply:
@sophokles73 : I agree to put the short version to the top, but we have a lot of disagreement regarding this point.
So I probably should wait for more comments regarding this?
@ppatierno @pellmann : can you please post a short comment to this point also? Thanks!

Reply:
after long discussion on chat we agreed on not having "auth-id" in the topic. I'm ok with that because it's part of the credentials which should not appear in the topic. Then the device could not know its device-id (former logical-id) so the only topic format we can use is the simple one "telemetry".

Reply:
For me the point of discussion here was if we need to separate between the device-id and a "logical-id" (solution-device-id) that is not known by the device. I thought it would be easier without this separation and clearer to always use the FQTA version - BUT if you want to stay with it, I think the short version is the natural one.

Reply:
Thanks to all - I read this as an agreement that I now put the STA version as main example and mention the FQTA as a note then.